### DISCUSSION TITLE
srsRAN_Project XnAP Handover

### DISCUSSION THREAD

**Discussion by Moussa-Guemdani** [srsran/srsRAN_Project#419] [General Help]
Hello, 
I am currently working on a project that aims to perform 5G handover through Xn Interface,

I would like to know if srsRAN_Project supports handover for 5G?

Thank you.

**Comment by pgawlowicz** [comment]
Intra-gNB handover is currently under development and will be released within the next months.

Please always check the set of available features and our roadmap here: https://docs.srsran.com/projects/project/en/latest/general/source/2_features_and_roadmap.html

**Reply by ikbalbtk** [reply]
hello, ther's any informations about inter-gNB handover ?

**Comment by richaaaaaaaa** [comment]
Hello,

Does the srsRAN project support Xn-based handover with the B210 USRP board?

I am currently working on this and have completed all the configurations as outlined in your document titled "srsRAN gNB Handover." However, when I physically move from one gNB to another, the TAC and PCI values change, but the handover message is not triggered.

Here are my setup details:

I am using a Samsung M14 COTS UE.
My setup includes an Open5GS 5G core and srsRAN 5G gNB.
In srsRAN, I have configured two different srsRAN gNBs using the B210 USRP board.
Please let me know if there is anything I might have missed.

Thank you.
